pkg/storaged/luksmeta-monitor-hack.py

```diff
@@ -42,6 +42,8 @@ def get_clevis_config_from_protected_header(protected_header):
         else:
             return {"pin": pin, pin: {}}
 
+    return None
```

Comment:
I think I'd prefer

```python
clevis = header_object.get('clevis')
if clevis is None:
    return None
```
...followed by the rest of the stuff, unindented

Reply:
Done.